js/utils.js

```diff
@@ -363,6 +363,16 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
     }
   }
 
+  function accessibilityDetails(event, item) {
+    if (event.key === 'Enter' || event.type === 'click') {
+      if (event.type === 'click') {
+        item.removeClass('focus-outline');
+      }
+      return true;
```

Comment:
Empty lines before and after `{ }` except when it's in the beginning or end of a `{ }` block.

```suggestion

      return true;
```

Reply:
Done